spec/views/job_runs/show.html.erb_spec.rb

```diff
@@ -24,6 +25,14 @@
       expect(rendered).to include("<a href=\"/job_runs/#{job_run.id}/download_report\">Download</a>")
     end
 
+    it 'with a completed with errors job, presents a download link to the report and the log file' do
+      job_run.started
+      job_run.completed_with_errors
+      render template: 'job_runs/show'
+      expect(rendered).to include("<a href=\"/job_runs/#{job_run.id}/download_log\">Download</a>")
+      expect(rendered).to include("<a href=\"/job_runs/#{job_run.id}/download_report\">Download</a>")
+    end
+
```

Comment:
another test to verify that a discovery job in a completed with errors state also shows the report link

Reply:
was it that it didn't show the link, or that the link was broken?